lib/iex/test/iex/autocomplete_test.exs

```diff
@@ -398,6 +398,14 @@ defmodule IEx.AutocompleteTest do
 
     assert expand('take(') == {:yes, '', ['take(enumerable, amount)']}
     assert expand('derive(') == {:yes, '', ['derive(protocol, module, options \\\\ [])']}
+
+    eval("defmodule NoDocs do; def sample(a), do: a; end")
```

Comment:
I don't think it needs to be define in the shell. I think  you can define a function in `IEx.AutocompleteTest` itself and try to call it from the shell.

Reply:
Indeed. Changed another thing and it looks a lot better now.